### PR TITLE
EX-2549 Address upstream comments

### PR DESCRIPTION
**Pull request by egsgordeev**
## Type of change

- [x] Bugfix
- [ ] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change
Address the comments from the upstream PR

### CODE REVIEW

**Comment by egsgordeev** [comment]
Relevant changes are only in `sovrnBidAdapter.js` and  `sovrnBidAdapter_spec.js`

**Review by jrosendahl — APPROVED** [review]
LGTM

File: modules/jixieBidAdapter.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ import { Renderer } from '../src/Renderer.js';
 export const storage = getStorageManager();
 
 const BIDDER_CODE = 'jixie';
-const EVENTS_URL = 'https://jxhbtrackers.azurewebsites.net/sync/evt?';
+const EVENTS_URL = 'https://hbtra.jixie.io/sync/hb?';
```
Comment (cpabst) [inline modules/jixieBidAdapter.js:12]: Where did this come from?

**Review by cpabst — APPROVED** [review]
LGTM

File: modules/jixieBidAdapter.js (hunk 1 above)
Comment (egsgordeev) [reply modules/jixieBidAdapter.js:12]: upstream origin :man_shrugging:

**Pull request merged**